apps/src/code-studio/pd/professional_learning_landing/landingPage.jsx

```diff
@@ -74,6 +75,11 @@ const LandingPage = React.createClass({
           />
         )
         }
+        {
+          <UpcomingWorkshops
+
+          />
```

Comment:
Why all the whitespace? If this is a placeholder for props you'll add in the next PR, cool. Otherwise please condense to `<UpcomingWorkshops />`